pkg/qmstr/service/common.go

```diff
@@ -33,3 +35,42 @@ func CreateErrorNode(errorMes string) *InfoNode {
 		Data: errorMes,
 	})
 }
+
+func (in *InfoNode) Describe(tab string) string {
+	describe := []string{fmt.Sprintf("%s|- Type: %s, Confidence score: %v", tab, in.Type, in.ConfidenceScore)}
+	tab = tab + "	"
```

Comment:
tab?

Reply:
It's used for output format purposes. So you can easily identify the connection, e.g. a node is derived from another one (the derived node has one more tab), etc.